libs/performance_counters/include/hpx/performance_counters/apex_sample_value.hpp

```diff
@@ -0,0 +1,43 @@
+//  Copyright (c) 2007-2020 Hartmut Kaiser
+//
+//  SPDX-License-Identifier: BSL-1.0
+//  Distributed under the Boost Software License, Version 1.0. (See accompanying
+//  file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+#if !defined(HPX_PERFORMANCE_COUNTERS_APEX_SAMPLE_VALUE_HPP)
+#define HPX_PERFORMANCE_COUNTERS_APEX_SAMPLE_VALUE_HPP
+
+#include <hpx/config.hpp>
+
+#ifdef HPX_HAVE_APEX
+
+#include <hpx/performance_counters/counters.hpp>
+#include <hpx/threading_base/external_timer.hpp>
+
+#include <cstdint>
+#include <string>
+
+namespace hpx { namespace util {
+
+    // The actual function pointers. Some of them need to be exported,
+    // because through the miracle of chained headers they get referenced
+    // outside of the HPX library.
+    static inline void sample_value(const std::string& name, double value)
+    {
+        if (sample_value_function != nullptr)
```

Comment:
Maybe?
```suggestion
        if (external_timer::sample_value_function != nullptr)
```

Reply:
Uhh, the whole code block belongs into the `namespace external_timer`, thanks for spotting this! Will fix asap.

Reply:
That should be fine now.